docs/content/docs/utilities/animations.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Animations
+description: PaperCSS Animations
+---
+
+### Effects:
+
+Ripple 
+<button class="ripple">Default ripple</button>
+
+Colored ripples
+<button class="ripple ripple-primary">Primary</button>
+<button class="ripple ripple-secondary">Secondary</button>
+<button class="ripple ripple-success">Success</button>
+<button class="ripple ripple-warning">Warning</button>
+<button class="ripple ripple-danger">Danger</button>
+
+#### Code:
+
+```html
+<p>Ripple</p>
+<button class="ripple">Default ripple</button>
+<p>Colored ripples</p>
+<button class="ripple ripple-primary">Primary</button>
+<button class="ripple ripple-secondary">Secondary</button>
+<button class="ripple ripple-success">Success</button>
+<button class="ripple ripple-warning">Warning</button>
+<button class="ripple ripple-danger">Danger</button>
+```
+
+### Animations
+
+<div class="row flex-spaces">
```

Comment:
For these animation examples, would it be possible for you to add a small amount of JS to make the animations interactive? Maybe just a button to add or remove the class names? It would be awesome to see the animations live on the site!